results/sentence-transformers__paraphrase-multilingual-MiniLM-L12-v2/bf3bf13ab40c3157080a7ab344c831b9ad18b5eb/NanoTouche2020Retrieval.json

```diff
@@ -0,0 +1,158 @@
+        "nauc_map_at_1000_max": 0.024631,
+        "nauc_map_at_1000_std": 0.029983,
+        "nauc_map_at_1000_diff1": -0.031561,
+        "nauc_recall_at_1_max": 0.661545,
+        "nauc_recall_at_1_std": 0.661545,
+        "nauc_recall_at_1_diff1": -0.423411,
+        "nauc_recall_at_3_max": -0.180809,
+        "nauc_recall_at_3_std": -0.053915,
+        "nauc_recall_at_3_diff1": 0.044132,
+        "nauc_recall_at_5_max": -0.135902,
+        "nauc_recall_at_5_std": -0.114599,
+        "nauc_recall_at_5_diff1": 0.071753,
+        "nauc_recall_at_10_max": -0.259895,
+        "nauc_recall_at_10_std": -0.305096,
+        "nauc_recall_at_10_diff1": 0.115453,
+        "nauc_recall_at_20_max": -0.120146,
+        "nauc_recall_at_20_std": -0.22471,
+        "nauc_recall_at_20_diff1": 0.081439,
+        "nauc_recall_at_100_max": 0.04975,
+        "nauc_recall_at_100_std": -0.28374,
+        "nauc_recall_at_100_diff1": 0.316647,
+        "nauc_recall_at_1000_max": 0.352838,
+        "nauc_recall_at_1000_std": -0.550299,
+        "nauc_recall_at_1000_diff1": 0.868984,
+        "nauc_precision_at_1_max": 0.661545,
+        "nauc_precision_at_1_std": 0.661545,
+        "nauc_precision_at_1_diff1": -0.423411,
+        "nauc_precision_at_3_max": -0.180809,
+        "nauc_precision_at_3_std": -0.053915,
+        "nauc_precision_at_3_diff1": 0.044132,
+        "nauc_precision_at_5_max": -0.135902,
+        "nauc_precision_at_5_std": -0.114599,
+        "nauc_precision_at_5_diff1": 0.071753,
+        "nauc_precision_at_10_max": -0.259895,
+        "nauc_precision_at_10_std": -0.305096,
+        "nauc_precision_at_10_diff1": 0.115453,
+        "nauc_precision_at_20_max": -0.120146,
+        "nauc_precision_at_20_std": -0.22471,
+        "nauc_precision_at_20_diff1": 0.081439,
+        "nauc_precision_at_100_max": 0.04975,
+        "nauc_precision_at_100_std": -0.28374,
+        "nauc_precision_at_100_diff1": 0.316647,
+        "nauc_precision_at_1000_max": 0.352838,
+        "nauc_precision_at_1000_std": -0.550299,
+        "nauc_precision_at_1000_diff1": 0.868984,
+        "nauc_mrr_at_1_max": 0.661545,
+        "nauc_mrr_at_1_std": 0.661545,
+        "nauc_mrr_at_1_diff1": -0.423411,
+        "nauc_mrr_at_3_max": 0.056482,
+        "nauc_mrr_at_3_std": 0.165241,
+        "nauc_mrr_at_3_diff1": -0.067174,
+        "nauc_mrr_at_5_max": 0.066802,
+        "nauc_mrr_at_5_std": 0.109115,
+        "nauc_mrr_at_5_diff1": -0.045084,
+        "nauc_mrr_at_10_max": 0.005711,
+        "nauc_mrr_at_10_std": 0.02243,
+        "nauc_mrr_at_10_diff1": -0.029471,
+        "nauc_mrr_at_20_max": 0.023335,
+        "nauc_mrr_at_20_std": 0.032696,
+        "nauc_mrr_at_20_diff1": -0.039557,
+        "nauc_mrr_at_100_max": 0.025951,
+        "nauc_mrr_at_100_std": 0.027453,
+        "nauc_mrr_at_100_diff1": -0.02839,
+        "nauc_mrr_at_1000_max": 0.024631,
+        "nauc_mrr_at_1000_std": 0.029983,
+        "nauc_mrr_at_1000_diff1": -0.031561,
+        "main_score": 0.11707,
```

Comment:
Why is performance very low compared to original announcement? https://www.zeta-alpha.com/post/fine-tuning-an-llm-for-state-of-the-art-retrieval-zeta-alpha-s-top-10-submission-to-the-the-mteb-be

Reply:
I'll compare with sentence transformers results